packages/native-pack-tool/src/platforms/android.ts

```diff
@@ -79,6 +81,26 @@ export class AndroidPackTool extends NativePackTool {
         return true;
     }
 
+    async openWithIde() {
+        const projPath = ps.join(this.params.buildDir, 'proj');
+        let batchFile = "studio.sh"
```

Comment:
Has this script file not been committed?

Reply:
I need to check logic of macos